xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/RecentlyUpdatedService.xml

```diff
@@ -216,8 +216,18 @@
   ## Add data to result
   #foreach ($property in $commentResponse.results)
     #if ($property.propertyname == 'date')
-      #set ($dateFormat = $datetool.getDateFormat("EEE MMM dd HH:mm:ss z yyyy", $datetool.getLocale(), $datetool.getTimeZone()))
-      #set ($date = $dateFormat.parse($property.propertyvalue_[0]))
+      ## We try to parse the date using the current locale. If if fails, we try with the 'en' locale. If that also
+      ## fails, the date won't be displayed.
```

Comment:
It might be a good idea to set the date to null first, otherwise you may reuse a previous value.